src/diffusers/loaders/unet.py

```diff
@@ -813,16 +817,29 @@ def _load_ip_adapter_weights(self, state_dict):
                 ).to(dtype=self.dtype, device=self.device)
 
                 value_dict = {}
-                for k, w in attn_procs[name].state_dict().items():
-                    value_dict.update({f"{k}": state_dict["ip_adapter"][f"{key_id}.{k}"]})
```

Comment:
This looks a bit like it's backwards breaking

Reply:
I don't think it's backward breaking as the `attn_procs` seems to be well constructed and well-guarded. The slow tests should reveal if it's BW breaking.

Reply:
I don't think it is backward breaking since we always convert and load the IP adapter weights on the fly and never provided a method to save it